src/Command/RunCommand.php

```diff
@@ -221,14 +225,16 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
 
         $this->installTestFrameworkIfNeeded($input, $output);
 
-        $this->consoleOutput = $this->getApplication()->getConsoleOutput();
+        $this->consoleOutput = new ConsoleOutput(new SymfonyStyle($input, $output));
```

Comment:
This allows to remove `Application::getConsoleOutput()`: it is not required per so but since it is used only here in `RunCommand`, there is no need to move that state up